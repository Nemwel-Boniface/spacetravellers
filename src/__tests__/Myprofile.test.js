import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configStore';
import MyProfile from '../components/myprofile/Myprofile'

it('This is atest to check the my profile page', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <MyProfile />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});