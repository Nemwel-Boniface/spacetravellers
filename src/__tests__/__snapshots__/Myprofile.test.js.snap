// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is atest to check the my profile page 1`] = `
<div
  className="myProfile"
>
  <section
    className="my-missions"
  >
    <h2>
      My Mission
    </h2>
    <ul
      className="reservedCont"
    >
      ;
    </ul>
  </section>
  <section
    className="my-rockets"
  >
    <h2>
      My Rockets
    </h2>
    <ul
      className="reservedCont"
    />
  </section>
</div>
`;
